astropy/modeling/tests/test_functional_models.py

```diff
@@ -478,9 +478,19 @@ def test_Voigt1D_norm(algorithm):
     """Test integral of normalized Voigt profile."""
     from scipy.integrate import quad
 
-    voi = models.Voigt1D(
-        amplitude_L=1.0 / np.pi, x_0=0.0, fwhm_L=2.0, fwhm_G=1.5, method=algorithm
-    )
+    def voigt(algorithm):
+        return models.Voigt1D(
+            amplitude_L=1.0 / np.pi, x_0=0.0, fwhm_L=2.0, fwhm_G=1.5, method=algorithm
+        )
+
+    if algorithm == "humlicek2":
+        with pytest.warns(
+            AstropyDeprecationWarning, match=r".* has been depricated .*"
```

Comment:
```suggestion
            AstropyDeprecationWarning, match=r"'humlicek' has been deprecated since .*"
```
Don't want to blanket-catch any deprecation warnings here.